cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
When switching the order and adding REQUIRED:
"opmcommon REQUIRED"
"opmparser"
- still fails to resolve symbols from opm-common which are required by opm-parser (OpmLog:: ......)